### ISSUE TITLE
Barcode are not read in big images

### ISSUE THREAD

**Issue by drentsch** [mubaidr/Javascript-Barcode-Reader#362]
As soon as I get a large image with a lot of other information the barcode contained in it is no longer recognized, I get a return code "???????????????????????".

![image](https://user-images.githubusercontent.com/3909661/72445539-26a0cc00-37b2-11ea-8fcd-e49d6df616c3.png)

Are there restrictions I'm not aware of?
I'm using Chrome-Browser on a Windows10

Thanks

**Comment by mubaidr** [comment]
Yes, please read: https://github.com/mubaidr/Javascript-Barcode-Reader/blob/master/README.md#known-issues

**Comment by joseph-animasds** [comment]
you need to crop it so that only the barcode is showing

**Comment by mubaidr** [comment]
I will add this feature to locate bar-code in the image.
